tools/base_labels/translation.py

```
@@ -0,0 +1,47 @@
+            {"role": "system", "content": label_prompt(source_lang, target_lang)},
+            {"role": "user", "content": json.dumps(labels, ensure_ascii=False, indent=2)},
+        ],
+        title="CircusWiki Base Label Localization",
+    )
+    try:
+        raw = strip_code_fences(chat_message_content(data, "base label translation")).strip()
+        translated = json.loads(raw)
+    except (KeyError, IndexError, json.JSONDecodeError) as exc:
+        raise RuntimeError(f"Base label translation response was not valid JSON: {data}") from exc
+    if not isinstance(translated, dict):
+        raise RuntimeError(f"Base label translation response must be a JSON object: {translated}")
```

Ruff (0.15.15)

[warning] Prefer `TypeError` exception for invalid type

(TRY004)